build.xml

```diff
@@ -98,6 +98,9 @@
         <exec command="${drush} site-install ${profile.machine} --yes --site-name='${profile.human}' --account-pass=admin --db-url=${db.url}" dir="${docroot}" passthru="true" />
         <chmod file="${site}" mode="0755" />
 
+        <!-- Install API Test. -->
+        <exec command="${drush} pm-enable api_test --yes" dir="${docroot}" passthru="true" />
```

Comment:
I don't think so. We really don't want people using this - specifically because of the non-uniqueness of the Client UUID. As discussed, we could provide a "starter" client for people in a separate (non MVP) issue.